src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1306,6 +1311,18 @@ private function addMessengerSection(ArrayNodeDefinition $rootNode, callable $en
                         ->then(static function (array $v): void { throw new InvalidConfigurationException(sprintf('The specified default bus "%s" is not configured. Available buses are "%s".', $v['default_bus'], implode('", "', array_keys($v['buses'])))); })
                     ->end()
                     ->children()
+                        ->arrayNode('stop_signals')
+                            ->beforeNormalization()
+                                ->ifString()->then(static fn (string $value): int => constant($value))
```

Comment:
shouldn't we check of the constant exists and is a supported signal?

Reply:
if constant doesn't exists - it will return null, and will be caught at validate. About supported signal, it's checked inside the Listeners itself.